app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -54,6 +54,7 @@ const props = defineProps<{
   nodePosition: Vec2
   navigator: ReturnType<typeof useNavigator>
   usage: Usage
+  closeIfClicked: (e: PointerEvent) => boolean
```

Comment:
I think the component's API would be simpler if instead a property like `associatedElements: HTMLElement[]` were passed; this would allow the component to handle more of its own closing logic, and avoid passing a property that is use non-reactively.

Reply:
Good suggestion, thanks. Applied it.